test/image-data.js

```diff
@@ -53,6 +53,14 @@ test("Build output path with prefix and suffix", t => {
     t.is(image.combineWithDirectory({directory: "d/e", prefix: "prefix-", suffix: "_suffix"}), "d/e/prefix-key_suffix.png");
 });
 
+test("Build output path with keep orignal extension", t => {
+    t.is(image.combineWithDirectory({directory: "d/e", keepExtension: true}), "d/e/key.png");
+});
+
+test("Build output path with suffix", t => {
```

Comment:
looks like "Build output path with suffix" is duplicated, did you mean to add a suffix test with an keep extension parameter?

Reply:
Ah! sorry this duplication is my miss-copy